features/guidance/turn-lanes.feature

```diff
@@ -290,25 +290,264 @@ Feature: Turn Lane Guidance
             | a,e       | road,through,through | depart,new name straight,arrive | ,1,   |
             | a,f       | road,right,right     | depart,turn right,arrive        | ,0,   |
 
-    Scenario: Anticipate Lane Change
+    @anticipate
+    Scenario: Anticipate Lane Change for subsequent multi-lane intersections
```

Comment:
I'd prefer to have a separate file for the base-case and the anticipation feature. In addition, you should try and handle special cases.

Collapsed instructions and roundabouts spring to mind.